### DISCUSSION TITLE
MQTT Data Update Frequency

### DISCUSSION THREAD

**Discussion by palorenz** [mr-manuel/venus-os_dbus-mqtt-battery#33] [General]
Hi, 

first of all thanks for all your work!

I have a general question regarding all your victron MQTT services:
Is there an optimal and/or a minimum MQTT data update frequency?
How often should the data on the dbus be updated?

Thanks for your advice
Patrick

**Comment by mr-manuel** [comment]
Hi, there is no minimum/maximum unless you don't hit the timeout, which you can change.

I would recommend a update rate of 1 s and not more than 5 s to have a stable system.

**Reply by palorenz** [reply]
Ok, thank you :-)
